gnucash/gnome/gnc-budget-view.c

```diff
@@ -840,6 +855,11 @@ gbv_get_accumulated_budget_amount(GncBudget* budget, Account* account, guint per
     info.total = gnc_numeric_zero();
     info.budget = budget;
     info.period_num = period_num;
+    info.pdb = gnc_pricedb_get_db (gnc_account_get_book (account));
+    info.total_currency = gnc_account_get_currency_or_parent(account);
+
```

Comment:
Only one blank line between the local variable block and the first block of code.